localstack/services/kms/models.py

```diff
@@ -201,8 +208,23 @@ def __init__(self, key_spec: str):
             crypto_serialization.PublicFormat.SubjectPublicKeyInfo,
         )
 
+    def load_material(self, material: bytes):
```

Comment:
we can rename this function to `load_key_material` so that it comes more better understanding

Reply:
Done!